.pre-commit-config.yaml

```diff
@@ -114,11 +114,18 @@ repos:
         pass_filenames: false
         language: script
         types: [text]
-        files: ^(homeassistant/.+/(manifest|strings)\.json|\.coveragerc|\.strict-typing|homeassistant/.+/services\.yaml|script/hassfest/.+\.py)$
+        files: ^(homeassistant/.+/(manifest|strings)\.json|\.coveragerc|homeassistant/.+/services\.yaml|script/hassfest/(?!mypy_config).+\.py)$
       - id: hassfest-metadata
         name: hassfest-metadata
         entry: script/run-in-env.sh python3 -m script.hassfest -p metadata
         pass_filenames: false
         language: script
         types: [text]
         files: ^(script/hassfest/.+\.py|homeassistant/const\.py$|setup\.cfg)$
+      - id: hassfest-mypy-config
```

Comment:
Can we have this not run if it's going to run `hassfest` anyway ?

Reply:
@epenet Mentioned something similar earlier [already](https://github.com/home-assistant/core/pull/65092#discussion_r794574258). I dismissed it initially, since both are only run for changes to `hassfest` files itself. With the last commit, I've updated the `files` filter to remove the remaining overlap.